moveit_planners/pilz_industrial_motion_planner/src/joint_limits_aggregator.cpp

```diff
@@ -115,7 +115,7 @@ void pilz_industrial_motion_planner::JointLimitsAggregator::updatePositionLimitF
       break;
     // LCOV_EXCL_START
     default:
-      ROS_ERROR_STREAM("Multi-DOF-Joints not supported. The robot won't move. " << joint_model->getName());
+      ROS_WARN_STREAM("Multi-DOF-Joint '" << joint_model->getName() << "' not supported.");
```

Comment:
This misses something we talked about in #2930: there still is no warning about virtual joints not being supported.

```suggestion
      ROS_WARN_STREAM("Multi-DOF or virtual joint '" << joint_model->getName() << "' is not supported.");
```

Reply:
This warning is not particularly about virtual joints, but about a (virtual) `floating` joint. Thus, I consider the proposed message sufficient.

Reply:
```suggestion
      ROS_WARN_STREAM("Floating joint '" << joint_model->getName() << "' is not supported.");
```

Reply:
I'm insistent because `Multi-ODF` is a joint type. That makes the message confusing.

Reply:
Multi-DOF isn't a joint type itself, but it comprises both, the `floating` and `planar` joint types. Thus, the message is perfectly valid. Your proposal is too narrow.